src/measure_theory/function/l1_space.lean

```diff
@@ -645,6 +649,33 @@ begin
   { exact hf.ennreal_to_nnreal },
 end
 
+lemma integrable_with_density_iff_integrable_coe_smul₀
```

Comment:
The name ending in `₀` is supposed to indicate that you use `ae_measurable` instead of measurable?
In my mind, that ending corresponds to lemmas using `null_measurable` (e.g. [measure_theory.ae_restrict_mem₀](https://leanprover-community.github.io/mathlib_docs/measure_theory/measure/measure_space.html#measure_theory.ae_restrict_mem%E2%82%80)), so it would be better to use something else.

Adding an `of_measurable` or `of_ae_measurable` to the already long names would not be a good solution. Maybe add `_ae`?

Same remark for the next lemma

Reply:
Yes, null_mesurable and ae_measurable are essentially the same thing, so using the same suffix ₀ makes sense to me. I can switch to `_ae` if you prefer, tell me!

Reply:
They are essentially the same thing indeed, but they're not the same `def`. I don't know which of those two considerations matters the most. ₀ is fine, I guess.